logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc

```diff
@@ -7,6 +7,55 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
+
+To get the status of the OpenShift Elasticsearch cluster, visit the Grafana dashboard in the *Monitoring* section of the {product-title} web console at
+`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`.
```

Comment:
I am not able to see eo_elasticsearch_cr_cluster_management_state, eo_elasticsearch_cr_restart_total fields under monitoring/dashboards/grafana-dashboard-cluster-logging. Are we sure this is the correct path? @rolfedh

Reply:
@igor-karpukhin what do you think?

Reply:
@sasagarw could you please post a link to the dashboard that you added with the https://issues.redhat.com/browse/LOG-1680?